core-addon/IonCore.js

```diff
@@ -7,8 +7,19 @@ const Storage = require("./Storage.js");
 // The path of the embedded resource used to control Ion options.
 const ION_OPTIONS_PAGE_PATH = "public/index.html";
 
+const ION_DEFAULT_ARGS = {
+  availableStudiesURI: "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/pioneer-study-addons-v1/records"
```

Comment:
Please add a comment stating that if this URL ever changes in production, the `manifest.json` file would need to be changed as well.